### PR TITLE
Update schedule

### PR DESCRIPTION
**Pull request by mxie**

### CODE REVIEW

**Comment by mxie** [comment]
@sherryseibel I know that you updated your slides with a more accurate schedule. Could you copy and paste there on here so that I can update this PR? Thanks!

**Comment by SherrySeibel** [comment]
Here is the updated schedule:
```
9:15 – Come in, eat breakfast
9:30 – Settle in
10:00 – Opening notes, introduction to sponsors and volunteers
10:10 – Intro to Ruby?
10:45– What is Ruby on Rails?
10:50 – What is a web app?
12:00 – LUNCH!
1:00 – Models, Scaffolding, and Migrations
1:30 – The Model-View-Controller (MVC) architecture
3:00 – BREAK!
3:15 – Navigations, links and routes
4:45 – Breakout session 1
5:15 – Closing Notes
5:30 – PARTAAAAYYY!!!!!!!
```

**Comment by mxie** [comment]
@suzicurran Mind taking a look at this and see if any changes should be made? Should be relatively close to how we did things last time, IIRC. Thanks in advance!

File: app/views/pages/schedule.html.erb [hunk 1]
```diff
@@ -14,28 +14,24 @@
...
       <tr>
-        <td>7:20pm</td>
-        <td colspan="2">What is Ruby? What is Ruby on Rails?</td>
+        <td>7:10pm</td>
+        <td colspan="2">Download and install software</td>
       </tr>
       <tr>
         <td>7:30pm</td>
-        <td colspan="2">Intro to the Command Line</td>
+        <td colspan="2">Presentation: Intro to the Command Line</td>
```
Comment (SViccari) [inline app/views/pages/schedule.html.erb]: I like the addition of "Presentation" 👍

**Comment by mxie** [comment]
Generalized some sections...check it out now:

schedule - railsbridge boston 2018-11-18 12-41-02

**Comment by pcai** [comment]
Hello from the future! I'm going to merge this in so we can tweak it further to reflect the schedule for the upcoming workshop (happening this week!)

**Pull request merged**